clusters/l3-sqnc/monitoring/deployment-alert.yaml

```diff
@@ -0,0 +1,45 @@
+---
+apiVersion: notification.toolkit.fluxcd.io/v1beta3
+kind: Provider
+metadata:
+  name: github-dispatch
+  namespace: monitoring
+spec:
+  type: githubdispatch
+  address: https://github.com/digicatapult/sqnc-flux-infra
+  secretRef:
+    name: github-dispatch
+---
+apiVersion: notification.toolkit.fluxcd.io/v1beta3
+kind: Alert
+metadata:
+  name: deployment-alert
+  namespace: monitoring
+spec:
+  providerRef:
+    name: github-dispatch
+  eventSeverity: info
+  eventSources:
+    - kind: HelmRelease
+      name: '*'
+      namespace: alice
+    - kind: HelmRelease
+      name: '*'
+      namespace: bob
+    - kind: HelmRelease
+      name: '*'
+      namespace: cert-manager
+    - kind: HelmRelease
+      name: '*'
+      namespace: flux-system
+    - kind: HelmRelease
+      name: '*'
+      namespace: keycloak
+    - kind: HelmRelease
+      name: '*'
+      namespace: kube-system
+    - kind: HelmRelease
+      name: '*'
+      namespace: monitoring
```

Comment:
In prod we use the namespaces `capacity`, `order` and `optimiser`

Reply:
changed them, also removed `kube-system`...